Update UnitConverterUNECERec20.js

RalphTro opened the pull request:
Completed mass units (thereby deliberately left out outdated uom values, e.g. "pfund", "slug", "ton, assay", "grain", etc.)

RalphTro committed:
Update UnitConverterUNECERec20.js

Completed mass units (thereby deliberately left out outdated uom values, e.g. "pfund", "slug", "ton, assay", "grain", etc.)

mgh128 reviewed (approved):
Thanks for adding these mass units

RalphTro committed:
Update UnitConverterUNECERec20.js

Added a couple of further mass units (e.g. pfund and slug). Though outdated, there may be use cases requiring them nevertheless.

RalphTro committed:
Merge pull request #3 from mgh128/master

Added units for molar_concentration
